Framework/Algorithms/src/MultipleScatteringCylinderAbsorption.cpp

```diff
@@ -34,22 +34,28 @@ using namespace Geometry;
 
 // Constants required internally only, so make them static. These are
 // Chebyshev expansion coefficients copied directly from Carpenter 1969 Table 1
+// clang-format off
 namespace { // anonymous
 static const double CHEBYSHEV[] = {
-    0.730284,  -0.249987, 0.019448,  -0.000006, 0.000249,  -0.000004, 0.848859,
-    -0.452690, 0.056557,  -0.000009, 0.000000,  -0.000006, 1.133129,  -0.749962,
-    0.118245,  -0.000018, -0.001345, -0.000012, 1.641112,  -1.241639, 0.226247,
-    -0.000045, -0.004821, -0.000030, 0.848859,  -0.452690, 0.056557,  -0.000009,
-    0.000000,  -0.000006, 1.000006,  -0.821100, 0.166645,  -0.012096, 0.000008,
-    -0.000126, 1.358113,  -1.358076, 0.348199,  -0.038817, 0.000022,  -0.000021,
-    0.0,       0.0,       0.0,       0.0,       0.0,       0.0,       1.133129,
-    -0.749962, 0.118245,  -0.000018, -0.001345, -0.000012, 1.358113,  -1.358076,
-    0.348199,  -0.038817, 0.000022,  -0.000021, 0.0,       0.0,       0.0,
-    0.0,       0.0,       0.0,       0.0,       0.0,       0.0,       0.0,
-    0.0,       0.0,       1.641112,  -1.241639, 0.226247,  -0.000045, -0.004821,
-    -0.000030, 0.0,       0.0,       0.0,       0.0,       0.0,       0.0,
-    0.0,       0.0,       0.0,       0.0,       0.0,       0.0,       0.0,
-    0.0,       0.0,       0.0,       0.0,       0.0};
+    // l= 0       1         2          3         4          5       // (m,n)
+    0.730284, -0.249987, 0.019448, -0.000006, 0.000249,  -0.000004, // (1,1)
+    0.848859, -0.452690, 0.056557, -0.000009, 0.000000,  -0.000006, // (1,2)
+    1.133129, -0.749962, 0.118245, -0.000018, -0.001345, -0.000012, // (1,3)
+    1.641112, -1.241639, 0.226247, -0.000045, -0.004821, -0.000030, // (1,4)
+    0.848859, -0.452690, 0.056557, -0.000009, 0.000000,  -0.000006, // (2,1)
+    1.000006, -0.821100, 0.166645, -0.012096, 0.000008,  -0.000126, // (2,2)
+    1.358113, -1.358076, 0.348199, -0.038817, 0.000022,  -0.000021, // (2,3)
+    0.0,      0.0,       0.0,      0.0,       0.0,       0.0,       // (2,4)
+    1.133129, -0.749962, 0.118245, -0.000018, -0.001345, -0.000012, // (3,1)
+    1.358113, -1.358076, 0.348199, -0.038817, 0.000022,  -0.000021, // (3,2)
```

Comment:
Assuming `(m,n) = (2,3) = (3,2)`. The other Chebyshev polynomials have this feature, but I wasn't sure and the paper isn't explicit about it.